extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -118,6 +137,30 @@ export class AmpAutocomplete extends AMP.BaseElement {
      */
     this.highlightUserEntry_ = false;
 
+    /**
+     * If the "inline" attribute is present on amp-autocomplete.
+     */
+    this.inline_ = false;
+
+    /**
+     * The value of the "inline" attribute on amp-autocomplete.
+     * @private {string}
+     */
+    this.trigger_ = '';
+
+    /**
+     * The base value obtained from the "src" attribute on amp-autocomplete.
+     * Used for creating static network endpoints. See generateSrc_.
+     * @private {string}
+     */
+    this.srcBase_ = '';
+
+    /**
+     * The value of the "query" attribute on amp-autocomplete.
+     * @private {string}
+     */
+    this.queryKey_ = '';
```

Comment:
`srcBase` and `queryKey` are not inline-only since it is a way to fetch data on every input without having to import `amp-bind`. I changed the `fetchIfEmail` back to `maybeFetch` to account for these two cases.

I will move `trigger` though, good point